guides/common/modules/ref_glossary-of-terms-used-in-project.adoc

```diff
@@ -0,0 +1,350 @@
+
+[[Discovery_image]]
+Discovery image:: Refers to the minimal operating system based on {EL} that is PXE-booted on hosts to acquire initial hardware information and to communicate with {ProjectServer} before starting the provisioning process.
+
+[[Discovery_plugin]]
+Discovery plugin:: Enables automatic bare-metal discovery of unknown hosts on the provisioning network.
+The plugin consists of three components: services running on {ProjectServer} and {SmartProxyServer}, and the Discovery image running on host.
+
+[[Discovery_rule]]
+Discovery rule:: A set of predefined provisioning rules which assigns a host group to discovered hosts and triggers provisioning automatically.
+
+[[Docker_tag]]
+Docker tag:: A mark used to differentiate container images, typically by the version of the application stored in the image.
+In the {ProjectWebUI}, you can filter images by tag under *Content* > *Docker Tags*.
+
+ifndef::satellite[]
+[[Enterprise_Linux]]
+Enterprise Linux:: An umbrella term for the following {RHEL}-like operating systems:
+
+* AlmaLinux
+* CentOS Linux
+* CentOS Stream
+* Oracle Linux
+* {RHEL}
+* Rocky Linux
+
+ifndef::orcharhino[]
+{Project} is tested on AlmaLinux and CentOS Stream.
+endif::[]
+endif::[]
+
+[[ERB]]
+ERB:: Embedded Ruby (ERB) is a template syntax used in provisioning and job templates.
+
+[[Errata]]
+Errata:: Updated RPM packages containing security fixes, bug fixes, and enhancements.
+In relationship to a host, erratum is *applicable* if it updates a package installed on the host and *installable* if it is present in the host's content view (which means it is accessible for installation on the host).
+
+[[External_node_classifier]]
+External node classifier:: A construct that provides additional data for a server to use when configuring hosts.
+{ProjectName} acts as an External Node Classifier to Puppet servers in a {Project} deployment.
++
+The External Node Classifier will be removed in a future {Project} version.
+
+[[Facter]]
+Facter:: A program that provides information (facts) about the system on which it is run; for example, Facter can report total memory, operating system version, architecture, and more.
+Puppet modules enable specific configurations based on host data gathered by Facter.
+
+[[Facts]]
+Facts:: Host parameters such as total memory, operating system version, or architecture.
+Facts are reported by Facter and used by Puppet.
+
+[[Foreman]]
+Foreman:: The component mainly responsible for provisioning and content lifecycle management.
+ifdef::satellite[]
+Foreman is the main upstream counterpart of {ProjectName}.
+endif::[]
+
+[[Foreman_Hook]]
+Foreman hook:: An executable that is automatically triggered when an orchestration event occurs, such as when a host is created or when provisioning of a host has completed.
+ifdef::satellite,orcharhino[]
++
+Foreman hook functionality is deprecated and will be removed in a future {Project} version.
+endif::[]
+
```

Comment:
```suggestion
```
Foreman Hook will be gone in 3.11.

Reply:
Tagging @adamlazik1 as the docs contact for removing foreman_hooks. I actually can't find any ticket to remove foreman_hooks from guides, I just found a ticket to cover this in RNs.

Reply:
https://github.com/theforeman/foreman-documentation/pull/2959

Reply:
Yeah the procedures will remain funcional as foreman_hooks is being replaced by foreman_webhooks. I'll get back to you on what changes to make in this glossary.